helpdesk_mgmt/demo/helpdesk_demo.xml

```diff
@@ -32,7 +32,7 @@
     </record>
 
     <record id="helpdesk_team_2" model="helpdesk.ticket.team">
-        <field eval="&quot;Helpesk&quot;" name="name"/>
+        <field eval="&quot;Helpdesk&quot;" name="name"/>
```

Comment:
This can be simplified as:
```suggestion
        <field name="name">Helpdesk</field>
```

Reply:
The whole file has the &quot; i will not change it.
Just the typo

Reply:
But you are changing that line, so fix it entirely. Other thing is that you don't touch that line.